### DISCUSSION TITLE
LAB 02.01 - Task 2: Money Change

### DISCUSSION THREAD

**Discussion by LadyRodas** [rramosp/ai4eng.v1#57] [02 - PYTHON]
Hola,
Estuve realizando la tarea 2. Money Change, y a pesar de que ya me funciona el código correctamente, al intentar enviar la respuesta para calificación muestra un error. 
Muestro el código y el error que me aparece.

![Codigo](https://user-images.githubusercontent.com/87790239/126579607-7ceaaa91-9950-4133-b7a0-8e16892b9a9c.PNG)
![Error](https://user-images.githubusercontent.com/87790239/126579608-0269801e-b840-4ccb-bcbb-f4090ad6276e.PNG)

**Comment by diegofavian94** [comment]
Hola, aquí la solución al problema es sencilla, estás haciendo un `print()` de la respuesta, recuerda que lo que debes hacer es un **return** es decir, en vez de `print(None)` o `print(r)` debería ser `return None` y `return r`.
